test/check-application

```diff
@@ -27,37 +27,66 @@ class TestApplication(testlib.MachineCase):
 
         # show image listing toggle
         b.wait_present('#containers-images tr:contains("busybox:latest")')
-        b.wait_visible('#containers-images tr:contains("busybox:latest")')
         b.click('#containers-images tbody tr:contains("busybox:latest") td.listing-ct-toggle')
         b.wait_present('#containers-images tbody tr:contains("busybox:latest") + tr button.btn-delete')
 
+        # make sure no running containers shown
+        b.click("#containers-containers-filter button")
+        b.wait_present("#containers-containers-filter .dropdown-menu")
+        b.click("#containers-containers-filter li[data-data='running'] a")
```

Comment:
This should use your shiny new `filter_continers()` function. Also, it's not strictly necessary as "running" ought to be the default setting. But it's ok to be explicit about it, too.